selfservice/form/fields.go

```diff
@@ -34,7 +34,7 @@ type Field struct {
 	// Pattern is the equivalent of `<input pattern="{{.Pattern}}">`
 	Pattern string `json:"pattern,omitempty"`
 
-	// Disabled is the equivalent of `<input disabled="{{.Disabled}}">`
+	// Disabled is the equivalent of `<input disabled>`
```

Comment:
This is actually correct, because `.Disabled` is `true` or `false`, leading to `<input disabled="true">` or `<input disabled="false">` which are both valid. This makes controlling the input field easy :)

```suggestion
	// Disabled is the equivalent of `<input disabled="{{.Disabled}}">`
```

Reply:
Actually I think I got it right, I should have explained it better. The [specification](https://developer.mozilla.org/it/docs/Web/HTML/Element/input#disabled) doesn't allow for a value to be specified. These three are all equivalent (they all produce a disabled input):
```html
  <input type="password" disabled="false">
  <input type="password" disabled="true">
  <input type="password" disabled>
```

Reply:
You're right! Would you be up for another patch to fix this? Thank you! :)

Reply:
https://github.com/ory/kratos/pull/486